bench/rust-analyzer-rs/optimized/y83zt9lwqltsa78.ll

```diff
@@ -549,8 +549,9 @@ define { i32, i32 } @_ZN9text_edit8TextEdit15apply_to_offset17ha87deef8c32a5b56E
   %10 = load i32, ptr %9, align 8, !noundef !16
   %11 = getelementptr inbounds i8, ptr %.sroa.0.02429, i64 28
   %12 = load i32, ptr %11, align 4, !noundef !16
-  %.not = icmp ult i32 %10, %1
-  br i1 %.not, label %13, label %.critedge
+  %.017 = tail call i8 @llvm.ucmp.i8.i32(i32 %10, i32 %1)
+  %switch = icmp ult i8 %.017, 2
```

Comment:
Duplicate of https://github.com/dtcxzyw/llvm-opt-benchmark/pull/1194#pullrequestreview-2238695484